examples/logical_interconnect_group_synergy.pp

```diff
@@ -35,11 +35,32 @@
   }
 }
 
+oneview_logical_interconnect_group{'Puppet FC LIG Synergy':
+  ensure => 'present',
+  data   => {
+    name               => 'Puppet FC LIG Synergy',
+    redundancyType     => 'Redundant',
+    enclosureIndexes   => [-1],
```

Comment:
According to oneview api, it should be `-1` or `1-5` and is mandatory. 
I´ve seen it fail with 1 before though. This should likely be something specified in the API documentation, but is unclear to me as well. 
If you have suggestions on comments to add here, please do :octocat: